net/strongswan/files/usr/lib/ipsec/_updown

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
What's wrong with the `./src/_updown/_updown` script in `$(PKG_INSTALL_DIR)`?

Reply:
As explained above, the script only works with iptables so far. But in the openwrt-master and openwrt-22.03 branch we have **nft** in use as the default firewall backend with `fw4`. The `_updown` script is a sample implementation from strongswan.
I extended it years ago in openwrt with this [patch](https://github.com/openwrt/packages/commit/6cd8fcabe6d1727192bf447c7adc8e1eb42ab8f7) so that we have a hotplug mechanism and don't always have to change the `_updown` script on every system upgrade. This is because it is overwritten with every system upgrade.

With this pull request, I want to change the `_updown` script so that we trigger a hotplug event in this script and then all scripts under `/etc/hotplug.d/ipsec` are processed. To ensure that the rules for the old iptables are also executed, I have moved the `iptables` stuff of the upstream `_updown` script for `iptables` to the `02-iptables`.
With this change, the `_updown` handling is modular and I can now work on the nft integration.
The nft commands will then be included in the 02-nft script, which I am writing just now.
It works as expected. I will update this pullrequest when I am ready with my change